website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,92 @@
+- **Scalable build-times**: allow building and deploying localized sites independently.
+- **Localize assets**: an image of your site might contain text that should be translated.
+- **No coupling**: not forced to use any SaaS, yet the integration is possible.
+- **Easy to use with [Crowdin](http://crowdin.com/)**: multiple Docusaurus 1 sites use Crowdin, and should be able to migrate to v2.
+- **Good SEO defaults**: setting useful SEO headers like [`hreflang`](https://developers.google.com/search/docs/advanced/crawling/localized-versions) for you.
+- **RTL support**: locales reading right-to-left (Arabic, Hebrew...) should be easy to use.
+- **Default translations**: theme labels are translated for you in [many languages](https://github.com/facebook/docusaurus/tree/master/packages/docusaurus-theme-classic/codeTranslations).
+
+## Showcase
+
+We worked with a few volunteers to **dogfood the i18n support** before releasing it.
+
+### Jest: upgrading Docusaurus
+
+An **existing Docusaurus 1 site** using i18n should be **able to upgrade to Docusaurus 2** and still be translated.
+
+**[Jest](https://jestjs.io)** was one of those sites, and we [**successfully upgraded it to Docusaurus 2**](https://jestjs.io/blog/2021/03/09/jest-website-upgrade).
+
+[![Jest i18n screenshot](/img/blog/2021-03-09-releasing-docusaurus-i18n/jest.png)](https://jestjs.io)
+
+### Redwood: adopting Docusaurus
+
+[Redwood](https://redwoodjs.com/) is a React full-stack Jamstack framework.
+
+They were looking for a solution to create an **internationalized learning platform**, and became early adopters of Docusaurus 2 i18n for [learn.redwoodjs.com](https://learn.redwoodjs.com/), using Crowdin to get it translated to French.
+
+[![Redwood i18n screenshot](/img/blog/2021-03-09-releasing-docusaurus-i18n/redwood.png)](https://learn.redwoodjs.com/)
+
+### Datagit: using Docusaurus LTR support
+
+The i18n system should work with any language, including **Right-to-Left languages**.
+
+[Datagit.ir](https://datagit.ir/) is using Farsi, and a simple configuration option is able to **flip the Docusaurus theme** to add the required Right-to-Left support.
+
+[![Datagit i18n screenshot](/img/blog/2021-03-09-releasing-docusaurus-i18n/datagit.png)](https://datagit.ir/)
+
+## Conclusion
+
+We sincerely hope you will adopt and like the new i18n support.
+
+This feature has not been an easy one, and we would like to thank everyone that got involved in this journey:
+
+- [Alexey](https://github.com/lex111) for his help to make the i18n support exhaustive and even better than v1.
+- [Simen](https://github.com/SimenB) for volunteering and supporting the Jest migration.
+- [Claire](https://github.com/clairefro) for adopting Docusaurus 2 on the new Redwood platform and providing many feedbacks.
+- [Massoud](https://github.com/massoudmaboudi) for reviewing my work on LTR and adopting it on Datagit.
+- [Crowdin](http://crowdin.com/) for their support and willingness to improve their translation SaaS.
```

Comment:
**CrowdIn** has a HTTPS domain, so probably should use `https` instead of `http` in the link.